core/schemas/observables/path.py

```diff
@@ -1,10 +1,21 @@
+import re
 from typing import Literal
 
 from core.schemas import observable
 
+LINUX_PATH_REGEX = re.compile(r"^(\/[^\/\0]+)+$")
+WINDOWS_PATH_REGEX = re.compile(
+    r"^(?:[a-zA-Z]\:|\\\\[\w\.]+\\[\w.$]+)\\(?:[\w]+\\)*\w([\w.])+"
+)
+
+
+def path_validator(value):
```

Comment:
Is an intermediate function really necessary here?

Reply:
not at all :)

Reply:
Let's inline it in is_valid then!

Reply:
fixed in 0dcd89b43fc45cbe350b64e44bd77d0fe6e0e1fe